Mathlib/GroupTheory/GroupAction/ConjAct.lean

```diff
@@ -259,6 +259,14 @@ theorem stabilizer_eq_centralizer (g : G) :
   le_antisymm (le_centralizer_iff.mp (zpowers_le.mpr fun _ => mul_inv_eq_iff_eq_mul.mp)) fun _ h =>
     mul_inv_eq_of_eq_mul (h g (mem_zpowers g)).symm
 
+theorem _root_.Subgroup.centralizer_eq_comap_stabilizer (g : G) :
+    Subgroup.centralizer {g} = Subgroup.comap ConjAct.toConjAct.toMonoidHom
+      (MulAction.stabilizer (ConjAct G) g) := by
+  ext k
+  simp only [mem_centralizer_iff, Set.mem_singleton_iff, forall_eq, ConjAct.toConjAct_smul]
+  rw [eq_comm]
```

Comment:
Can you check (quickly) if the need for `rw [eq_comm]` is caused by a lemma written in a way different wrt to the others?

Reply:
By `mem_centralizer_iff`, k belongs to the centralizer of {g} means g * k = k * g, while k belongs to the stabilizer of `g` by the conjugation action means that `k * g * k^{-1} = g`.  
It probably makes sense to rephrase `mem_centralizer_iff` in the opposite direction. (But please let
me do that elsewhere!)

Reply:
I let a note that maybe we should do that sometime.